.pre-commit-config.yaml

```diff
@@ -0,0 +1,17 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
+    hooks:
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: check-yaml
+      - id: check-json
+
```

Comment:
could you add 
* check-added-parge-files
* check-merge-conflict
* check-toml

Reply:
```suggestion
      - id: check-added-large-files
      - id: check-merge-conflict
      - id: check-toml      
```